### PR TITLE
Addresses #75 - Remove unessecary units conversion in TethysComms

### PR DESCRIPTION
**Pull request by arjo129**
Depends on ignitionrobotics/ign-gazebo#1269.

This PR attempts to remove unnecessary unit conversions in tethys from angular velocity -> force -> angular velocity.

Signed-off-by: Arjo Chakravarty <arjo@openrobotics.org>

### CODE REVIEW

File: lrauv_ignition_plugins/src/TethysCommPlugin.cc
Comment (chapulina) [inline lrauv_ignition_plugins/src/TethysCommPlugin.cc:417]: We should also change `thrusterTopic` to `cmd_vel`, otherwise it's treating this as angular velocity

```diff
diff --git a/lrauv_ignition_plugins/src/TethysCommPlugin.hh b/lrauv_ignition_plugins/src/TethysCommPlugin.hh
index 1b18ae4..8eecae4 100644
--- a/lrauv_ignition_plugins/src/TethysCommPlugin.hh
+++ b/lrauv_ignition_plugins/src/TethysCommPlugin.hh
@@ -106,7 +106,7 @@ namespace tethys
 
     /// Topic to publish to for thruster
     private: std::string thrusterTopic
-      {"propeller_joint/cmd_pos"};
+      {"propeller_joint/cmd_vel"};
 
     /// Topic to publish to for rudder
     private: std::string rudderTopic
```

**Review by chapulina — CHANGES_REQUESTED** [review]
Did you notice that propelling backwards always makes the vehicle rotate, but propelling forward it goes straight? It's probably unrelated to this PR.

**Comment by arjo129** [comment]
Thats expected behavior. The hydrodynamic model is not stable in reverse and is not an issue since the vehicle does not operate in reverse.

**Review by chapulina — APPROVED** [review]

**Comment by chapulina** [comment]
We need to wait for upstream to be merged and the tests to pass before merging this one

**Comment by chapulina** [comment]
Upstream is merged and released, retriggered CI :crossed_fingers:

**Pull request merged**
